### ISSUE TITLE
Program quit exceptionally after executing the function of transforms.Normalize((0.5,), (0.5,))

### ISSUE THREAD

**Issue by songyuc** [pytorch/vision#2054]
## 🐛 Bug

Program quit exceptionally at executing  the function of transforms.Normalize((0.5,), (0.5,))

## To Reproduce
Reproducing it needs my data.

1. Loaded my data into im0 as np.ndarray [28,28]
2. Expanding the dimensions of im0 as `im1 = np.expand_dims(im0, axis=2)`
3. Getting a deep copy as `a = im1.copy()`
4. Transformed it into a torch.Tensor as
```
import torchvision.transforms as transforms
f1 = transforms.ToTensor()
im2 = f1(a)
im2 = im2.float()
im2.shape
```
5. Then did the normalizing as
```
f2 = transforms.Normalize((0.5,), (0.5,))
im3 = f2(im2)
im3.shape
```
and the program quitted exceptionally.

![image](https://user-images.githubusercontent.com/27288110/71084238-cd07aa80-21cf-11ea-913e-bc60b6592958.png)

And I have worked out the reproduction code (please neglect the Chinese comments, because I am a student from China):
```Python

import numpy as np

a = np.random.randint(0, 255, (28, 28),dtype=np.int64)

a.shape

 

# Out[1]:

# (28, 28)

 

# 增加维度Channel

im1 = a

im1 = np.expand_dims(im1, axis=2)

im1.shape

 

# Out[2]:

# (28, 28, 1)

 

# 进行.ToTensor()的变换

import torchvision.transforms as transforms

f1 = transforms.ToTensor()

im2 = f1(im1)

im2.shape

 

# Out[3]:

# torch.Size([1, 28, 28])

 

# 获得规范化的变换

f2 = transforms.Normalize((0.5,), (0.5,))

f2

 

# Out[4]:

# Normalize(mean=(0.5,), std=(0.5,))

 

# 对数据进行规范化处理

im3 = f2(im2)

print("Finish")
# （没有输出，并且Notebook提示“Dead kernel”，也就是核进程崩溃了）
```
And there's no output, even like "Finish", and the Notebook would show “Dead kernel”, which means the Kernel process have broken.

## Expected behavior

The program quitted as,
![image](https://user-images.githubusercontent.com/27288110/71084238-cd07aa80-21cf-11ea-913e-bc60b6592958.png)

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0): 1.3.1
 - OS (e.g., Linux): Win10
 - How you installed PyTorch (`conda`, `pip`, source): conda
 - Build command you used (if compiling from source): by PyCharm
 - Python version: 3.7.3
 - CUDA/cuDNN version: CUDA 10.1
 - GPU models and configuration: using CPU
 - Any other relevant information:

## Additional context

cc @ezyang @gchanan @zou3519 @fmassa @peterjc123

**Comment by fmassa** [comment]
@songyuc @gen2020code `Normalize` should be called on float tensors after `ToTensor`. If your tensor is not floating point, then normalizing by `0.5` (which gets rounded to 0) doesn't make sense and you have an issue in your code.

The sigterm is unfortunate, but is outside of the scope of torchvision.
